packages/generator-helper/src/types.ts

```diff
@@ -75,13 +76,16 @@ export type BinaryPaths = {
 
 export type GeneratorOptions = {
   generator: GeneratorConfig
+  // TODO: what is otherGenerators for?
   otherGenerators: GeneratorConfig[]
   schemaPath: string
   dmmf: DMMF.Document
   datasources: DataSource[]
+  // TODO deprecate datamodel & rename to schema?
   datamodel: string
-  binaryPaths?: BinaryPaths
+  // TODO is it really always version hash? Feature is unclear.
```

Comment:
probably engine version, so git commit hash?

Reply:
It seems we use it for something and third party users use it for something else